notebooks/30_conditionals/conditionals_howto.ipynb

```diff
@@ -4,36 +4,31 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "  # How-Tos für Vergleichsoperatoren und bedingte Anweisungen\n",
-    "  ## Vergleichsoperatoren\n",
-    "  * [List der Vergleichsoperatoren](#List-der-Vergleichsoperatoren)"
-   ]
-  },
-  {
-   "cell_type": "markdown",
-   "metadata": {},
-   "source": [
-    "  ## Bedingte Anweisungen\n",
-    "  * [Einfachses if](#Einfaches-if)\n",
-    "  * [if-else](#if-else)\n",
-    "  * [if-elif-else](#if-elif-else)\n",
-    "  * [Komplexe Bedingungen](Komplexe-Bedingungen)"
+    "# How-tos For Comparison Operators And Conditional Statements\n",
```

Comment:
You changed the German howto to English. This should not happen. We need a separate English howto file.

Reply:
fixed